### ISSUE TITLE
Add FAQs section

### ISSUE THREAD

**Issue by yatri1609** [fossasia/badge-magic-android#76]
**Describe the feature you'd like**
A simple FAQs section which will guide users with simple questions.

**Additional context**
Dependent on #52 

**Would you like to work on the issue?**
YES

**Comment by yashk2000** [comment]
@yatri1609 since you have created so many issues related to the nav drawer, and have already made the nav drawer, go ahead and implement all these features. After you do that, I'll add the about part. Is that alright?

**Comment by yatri1609** [comment]
Cool no problem. Thanks :)

**Comment by mariobehling** [comment]
Low priority. Please focus on moving forward the app through features first:
https://github.com/fossasia/badge-magic-android/labels/Priority%3A%20High

Closing this therefore.
